web/assets/css/main.css

```diff
@@ -236,6 +236,15 @@ input[type='checkbox']:checked + label {
     box-shadow: 0 0 0 1px #ffffff60 !important;
 }
 
+.timer {
+    display:flex;
+    font-variant-numeric: tabular-nums; /* Otherwise the counter will "jump" */
+}
+
+.timer span + span:before {
+    content:":"
+}
+
```

Comment:
You can use the classes `flex`, `tabular-nums` and `after:content-[':']`(?) (https://tailwindcss.com/docs)

Reply:
Thanks, the first two work. However, `after:content-[':']` does not work from my testings. My solution would be to just insert the `:` manually in the template, I think using CSS for this is not necessary.